### DISCUSSION TITLE
Receiving null dates in the `commit.author` or `commit.committer` fields of `CommitResponse`

### DISCUSSION THREAD

**Discussion by mdblr** [octokit/octokit.js#2731] [Q&A]
Hello,

I'm trying to understand the conditions for which the `CommitResponse` object would have null dates in the `commit.author.date` or `commit.committer.date` fields.

It would be helpful in order to make a contingency plan (e.g. retrying the call vs exploring default options).

But, I could not find any information in the documentation. I figured this would be a good first place to ask if the library understands that it may be null. 

TYIA for the help.

**Comment by wolfy1339** [comment]
Your best bet is to ask GitHub directly. We don't really know more than you.

GitHub publishes an OpenAPI schema of their API and we use that to generate the types for the different endpoints.
They also publish a GraphQL schema that we consume and generate types from

**Reply by lulitzarodgarr** [reply]
_ import { Octokit } from "octokit";

const octokit = new Octokit({ });

const data = await octokit.paginate("GET /repos/{owner}/{repo}/issues", {
  owner: "octocat",
  repo: "Spoon-Knife",
  per_page: 100,
  headers: {
    "X-GitHub-Api-Version": "2022-11-28",
  },
});

console.log(data)
_
